python/tank/deploy/tank_commands/core_localize.py

```diff
@@ -56,7 +60,7 @@ def run_noninteractive(self, log, parameters):
         """
         pc_root = self.tk.pipeline_configuration.get_path()
         log.debug("Executing the localize command for %r" % self.tk)
-        return do_localize(log, pc_root, suppress_prompts=True)
+        return do_localize(log, pc_root, suppress_prompts=True, strip_toolkit_credentials=False)
```

Comment:
Personal preference, but I would have named this strip_script_user_credentials. You be the judge.